### DISCUSSION TITLE
How to specify clang format version 12?

### DISCUSSION THREAD

**Discussion by fnc12** [DoozyX/clang-format-lint-action#33] [General]
I am developing sqlite_orm lib and I want to update clang-format to 12. I've [changed version of your repo](https://github.com/fnc12/sqlite_orm/pull/793/commits/7b7e47bb5106486158c294c51555a867edd170f1) but I am still getting the same error: code is formatted differently (I have clang-format 12 installed on my machine). https://github.com/fnc12/sqlite_orm/runs/3723489510?check_suite_focus=true
<img width="1059" alt="Снимок экрана 2021-09-27 в 20 34 25" src="https://user-images.githubusercontent.com/6009640/134957734-62d07ec1-7dc5-4b72-b685-65ed283e1f80.png">
I'd like to make github actions to perform code formatting check using clang-format 12. Thanks

**Comment by DoozyX** [comment]
Hello, just saw your changes, I think you missed the clangFormatVersion option, it should be like this
```
    - uses: DoozyX/clang-format-lint-action@v0.12
      with:
        clangFormatVersion: 12
 ```

**Reply by fnc12** [reply]
thanks. It worked
